torch/_inductor/runtime/triton_heuristics.py

```diff
@@ -823,6 +827,17 @@ def restore_args_from_cpu(self, cpu_copies):
             arg, cpu_arg = pair
             arg.copy_(cpu_arg, non_blocking=True)
 
+    def reset_to_zero_args(self, *args, **kwargs):
```

Comment:
Can we early exit `if not self.reset_to_zero_arg_names`? Otherwise this will slow down the default code path.